src/app/routes/regex/utils/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,17 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Hmmm think I'd prefer a expectance test than a snapshot but won't block the PR on this